history/characters/wc_easteregg_characters.txt

```diff
@@ -457,4 +457,267 @@ easteregg13={ #Shrek reference
 		}
 	}
 	688.1.1={ death=yes }
-}
\ No newline at end of file
+}
+
+### TALES OF THE PAST MACHINIMA CHARACTERS ###
+
+# Part of Eden Aurorae guild, therefore a landless playable character
+easteregg14 = {
+	name = "Blazer"
+	dna = blazer_dna
+	culture = azerothian
+	religion = holy_light
+	disallow_random_traits = yes
+	#Stats
+	diplomacy = 5
+	martial = 5
+	stewardship = 4
+	intrigue = 7
+	learning = 4
+	#Personality
+	trait = brave
+	trait = gregarious
+	trait = stubborn
+	#Education
+	trait = education_intrigue_2
+	#Other
+	trait = adventurer
```

Comment:
keep in mind that once Roads to Power comes out this trait will be renamed to Veteran Adventurer for those who used to be adventurers

Reply:
noted